core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -68,6 +69,9 @@ export class OpportunitiesListItemComponent {
   cardsAvailable: number = 0;
   onMobile!: boolean;
   resizeSubscription!: Subscription;
+  translationLengthLabel: string = '';
+  translationLengthLabelRequired: boolean = false;
+  translationLengthInWords: number = 0;
```

Comment:
Doesn't seem necessary to create a new variable for this. We can just access it from the opportunity in the html.